include/logging/log_ctrl.h

```diff
@@ -170,6 +170,15 @@ void log_backend_enable(struct log_backend const *const backend,
  */
 void log_backend_disable(struct log_backend const *const backend);
 
+/**
+ * @brief Get backend by name.
+ *
+ * @param[in] backend_name Name of the backend as defined by the LOG_BACKEND_DEFINE.
+ *
+ * @retval Pointer to the backend instance if found, NULL if backend is not found.
+ */
+extern const struct log_backend *log_backend_get_by_name(const char *backend_name);
```

Comment:
why do we need the extern here?

Reply:
I think we do not need it as such. I can remove the extern.